bundles/org.openhab.binding.wiz/src/main/java/org/openhab/binding/wiz/internal/WizBindingConstants.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.wiz.internal;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link WizBindingConstants} class defines common constants, which
+ * are used across the whole binding.
+ *
+ * @author Sriram Balakrishnan - Initial contribution
+ * @author Joshua Freeman - update version
+ */
+@NonNullByDefault
+public class WizBindingConstants {
+
+    /**
+     * The binding id.
+     */
+    public static final String BINDING_ID = "wiz";
+
+    /**
+     * List of all Thing Type UIDs.
+     */
+    public static final ThingTypeUID THING_TYPE_COLOR_BULB = new ThingTypeUID(BINDING_ID, "color-bulb");
+    public static final ThingTypeUID THING_TYPE_TUNABLE_BULB = new ThingTypeUID(BINDING_ID, "tunable-bulb");
+    public static final ThingTypeUID THING_TYPE_DIMMABLE_BULB = new ThingTypeUID(BINDING_ID, "dimmable-bulb");
+    public static final ThingTypeUID THING_TYPE_SMART_PLUG = new ThingTypeUID(BINDING_ID, "plug");
+    public static final ThingTypeUID THING_TYPE_CEILING_FAN = new ThingTypeUID(BINDING_ID, "ceiling-fan");
+
+    /**
+     * The supported thing types.
+     */
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Stream.of(THING_TYPE_COLOR_BULB,
+            THING_TYPE_TUNABLE_BULB, THING_TYPE_DIMMABLE_BULB, THING_TYPE_SMART_PLUG, THING_TYPE_CEILING_FAN)
+            .collect(Collectors.toSet());
+
+    /**
+     * List of all Channel ids
+     */
+    public static final String CHANNEL_BRIGHTNESS = "brightness";
+    public static final String CHANNEL_COLOR = "color";
+    public static final String CHANNEL_LAST_UPDATE = "last-update";
+    public static final String CHANNEL_MODE = "mode";
+    public static final String CHANNEL_REVERSE = "reverse";
+    public static final String CHANNEL_RSSI = "signal-strength";
```

Comment:
Since the RSSI is mapped to signal strength with values 0-5 and the name of the channel is also `signal-strength`, this constant carries a misleading name.
```suggestion
    public static final String CHANNEL_SIGNAL_STRENGTH = "signal-strength";
```